bugsnag-android-core/src/test/java/com/bugsnag/android/EventTest.java

```diff
@@ -4,71 +4,49 @@
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertTrue;
 
 import org.junit.Before;
 import org.junit.Test;
 
-import java.lang.Thread;
+import java.util.Collections;
 import java.util.Map;
 
 @SuppressWarnings("unchecked")
 public class EventTest {
 
+    private final HandledState handledState
+            = HandledState.newInstance(HandledState.REASON_HANDLED_EXCEPTION);
     private ImmutableConfig config;
     private Event event;
 
     /**
      * Generates a new default event for use by tests
      *
-     * @throws Exception if initialisation failed
      */
     @Before
-    public void setUp() throws Exception {
+    public void setUp() {
         config = BugsnagTestUtils.generateImmutableConfig();
         RuntimeException exception = new RuntimeException("Example message");
-        event = new Event.Builder(config, exception, null,
-            Thread.currentThread(), false, new Metadata()).build();
+        HandledState handledState = this.handledState;
+        event = new Event(exception, config, handledState);
     }
 
     @Test
-    public void checkExceptionMessageNullity() throws Exception {
```

Comment:
It appears the exception message can still be null - isn't the test still valid?

Reply:
Added this test case back in.